api/python/cell_census/setup.cfg

```diff
@@ -16,19 +17,18 @@ classifiers =
     Topic :: Scientific/Engineering :: Bio-Informatics
     Operating System :: POSIX :: Linux
     Operating System :: MacOS :: MacOS X
-    Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
     Programming Language :: Python :: 3.10
 
 [options]
-python_requires = >= 3.7, < 3.11
+python_requires = >= 3.8, < 3.11
 install_requires =
     numba
     numpy
     requests
-    tiledb>=0.18.1
-    tiledbsoma>=0.5.*
+    tiledb>=0.19.0
+    tiledbsoma==0.5.0a2
```

Comment:
Did you want `0.2.0rc0` now? (per PR summary)

Reply:
summary was incorrect - now fixed